File: src/main/java/logan/pickpocket/main/PickpocketUtils.java
package logan.pickpocket.main;

import logan.pickpocket.config.Config;
import org.bukkit.Material;

/**
 * Shared utility helpers for pickpocket rules.
 */
public final class PickpocketUtils {

    private PickpocketUtils() {
    }

    /**
     * @param type item material
     * @return true when item type is disallowed by config
     */
    public static boolean isItemTypeDisabled(Material type) {
        return Config.getDisabledItems().contains(type.name().toLowerCase());
    }
}

File: src/logan/pickpocket/main/PickpocketUtils.kt
package logan.pickpocket.main

import logan.pickpocket.config.PickpocketConfiguration
import org.bukkit.Material

class PickpocketUtils {
    companion object {
        fun isItemTypeDisabled(type: Material) =
            PickpocketConfiguration.disabledItems.contains(type.name.lowercase())
    }
}